lib/stale.js

```diff
@@ -82,6 +83,11 @@ module.exports = class Stale {
     queryParts.push(exemptMilestones ? 'no:milestone' : '')
     queryParts.push(exemptAssignees ? 'no:assignee' : '')
 
+    /** Can't search issues with draft:false in the query so this is appended only for pulls **/
+    if (type === 'pulls') {
+      queryParts.push(exemptDrafts ? 'draft:false' : '')
+    }
```

Comment:
I didn't write tests for this yet, but if this PR is agreeable I will definitely add those.

As for why I conditionally wrapped it - if you include `draft:boolean` in your query for issues, nothing returns - so this can only be applied for pulls.